R/vdiagram-data.R

```diff
@@ -63,21 +79,15 @@ historyMeasurementsLabel <- function(reportObject) {
   }
 }
 
+#' Set default historic flags
+#' @description Sets the historic flag to false if " " is passed in.
+#' @param histFlag The historic flag field from a V Diagram.
+#' @return FALSE if " " is passed in or if the field is empty, otherwise it returns what was passed in. 
 defaultHistFlags <- function(histFlag){
-  x <- NULL
-  if (length(histFlag)==1 && histFlag == " "){
-    histFlag <- rep(TRUE, length(x))
-  }
-
-  if (isEmptyOrBlank(histFlag)){
+  
+  if (isEmptyOrBlank(histFlag) || (length(histFlag)==1 && histFlag == " ")){
     histFlag <- FALSE
   }
```

Comment:
Yeah, that was a weird piece of logic. It did that, then changed it to false. I assume it's to make " " become FALSE because empty strings would look bad? The length==1 seems to be to check that it isn't null?